.gitignore

```diff
@@ -3,3 +3,4 @@ lib/
 types/
 node_modules/
 .vscode/
+coverage
```

Comment:
What produces a `coverage` file (or directory) that required this?

Reply:
Running the coverage report using jest (`npx jest --coverage`). I undid the change.